sql_alchemy_project8/project8/project8/sqlalchemy_proj8.py

```diff
@@ -0,0 +1,261 @@
+from sqlalchemy.orm import Session, Mapped, declarative_base, relationship, mapped_column
+from sqlalchemy import create_engine, insert, Index, select, Integer, Column, String, PrimaryKeyConstraint, ForeignKey
+from typing import Optional
+from sqlalchemy.exc import IntegrityError
+import pandas as pd
+from dotenv import load_dotenv
+import os
+import logging
+
+load_dotenv()
+
+PATH_ERROR = os.getenv("PATH_ERROR")
+POSTGRES_KEY = os.getenv("POSTGRES_KEY")
+PATH_DATA = os.getenv("PATH_DATA")
+ARTWORK_CSV = os.getenv('ARTWORK_CSV')
+
+logging.basicConfig(filename=PATH_ERROR, level=logging.ERROR,
+                    format='%(levelname)s (%(asctime)s) : %(message)s (%(lineno)d)')
+
+# columns to be loaded in from data
+columns = ["DisplayName", "ArtistBio", "Nationality", "Gender", "BeginDate", "EndDate"]
+artist_df = pd.read_csv('Artists.csv', usecols=[col for col in columns])
+
+columns2 = ['Artist', 'ConstituentID', 'ArtistBio', 'Nationality',
+            'Gender', 'Medium', 'Dimensions', 'CreditLine',
+            'AccessionNumber', 'Classification', 'Department',
+            'Cataloged', 'ObjectID']
+artwork_df = pd.read_csv(ARTWORK_CSV, usecols=[col for col in columns2])
+
+
+# connecting to database
+def create_database_engine(postgres_key):
+    '''
+    this function creates a database engine
+    '''
+    try:
+        database_engine = create_engine(postgres_key, echo=True)
+        logging.info("Database engine created successfully.")
+        return database_engine
+    except Exception as e:
+        logging.info(f"Error creating database engine: {e}")
+        return None
+
+
+engine = create_database_engine(POSTGRES_KEY)
+
+Base = declarative_base()
+
+
+# creating table in database
+
+class Artist(Base):
+    __tablename__ = "Artist"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    DisplayName: Mapped[str] = mapped_column(primary_key=True)
```

Comment:
Do these CamelCases later translate for snake_cases? If not lower case and snake_cases please